src/LowLevelMotionController.cpp

```diff
@@ -251,6 +251,8 @@ int main(int argc, char **argv)
 
     ros::Rate rate (update_frequency);
 
+    iarc7_msgs::OrientationThrottleStamped last_uav_command;
```

Comment:
It looks like you rebased and managed to include a commit that shouldn't be here.